.gitignore

```diff
@@ -48,3 +48,6 @@ yarn-error.log*
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.vscode/launch.json
+.vscode/settings.json
```

Comment:
Have some pretty specific launch configuration, and don't want to potentially collide with anything you have!